drivers/overlay/ov_network.go

```diff
@@ -298,6 +298,14 @@ func (n *network) initSubnetSandbox(s *subnet) error {
 	return nil
 }
 
+func (n *network) cleanupStaleSandboxes() {
+	for i := 1; i < n.initEpoch; i++ {
```

Comment:
Trying to understand the logic. Since the `initEpoch` is not marshalled, I guess that on every daemon restart and first time the network is used, `initEpoch` will always be set to 1. How will this cleanup be useful if the `initEpoch` is not saved ?